staff/pedro-garriganogues/ecomerce/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE <!DOCTYPE html>
+<html>
```

Comment:
Try to change all the ids that you use inside the tags, for classes. You have to try to not use ids, and let them only when it's necessary.